pipelines/coaddDiffMatchedInjectedCore.yaml

```diff
@@ -0,0 +1,201 @@
+#      atools.matchedRefPositionYDiff.produce.plot.yLims: lims_pos_diff_full
+#      atools.matchedRefPositionYDiff.scale_factor: parameters.scale_pixel
+#
+#      atools.matchedRefPositionYChi: MatchedRefCoaddDiffPositionTool
+#      atools.matchedRefPositionYChi.coord_meas: y
+#      atools.matchedRefPositionYChi.coord_ref: refcat_y
+#      atools.matchedRefPositionYChi.produce.plot.xLims: lims_mag_x_full
+#      atools.matchedRefPositionYChi.produce.plot.yLims: lims_pos_chi
+#      atools.matchedRefPositionYChi.compute_chi: true
+
+      atools.matchedRefPositionRaDiff: MatchedRefCoaddDiffPositionTool
+      atools.matchedRefPositionRaDiff.coord_meas: coord_ra
+      atools.matchedRefPositionRaDiff.coord_ref: ref_ra
+      atools.matchedRefPositionRaDiff.produce.plot.xLims: lims_mag_x_full
+      atools.matchedRefPositionRaDiff.produce.plot.yLims: lims_pos_diff_full
+
+      atools.matchedRefPositionRaDiffZoom: MatchedRefCoaddDiffPositionTool
+      atools.matchedRefPositionRaDiffZoom.coord_meas: coord_ra
+      atools.matchedRefPositionRaDiffZoom.coord_ref: ref_ra
+      atools.matchedRefPositionRaDiffZoom.produce.plot.xLims: lims_mag_x_zoom
+      atools.matchedRefPositionRaDiffZoom.produce.plot.yLims: lims_pos_diff_zoom
+      atools.matchedRefPositionRaDiffZoom.produce.metric: NoMetric
+
+      atools.matchedRefPositionRaChi: MatchedRefCoaddDiffPositionTool
+      atools.matchedRefPositionRaChi.coord_meas: coord_ra
+      atools.matchedRefPositionRaChi.coord_ref: ref_ra
+      atools.matchedRefPositionRaChi.coord_ref_cos: ref_dec
+      atools.matchedRefPositionRaChi.produce.plot.xLims: lims_mag_x_full
+      atools.matchedRefPositionRaChi.produce.plot.yLims: lims_pos_chi
+      atools.matchedRefPositionRaChi.compute_chi: true
+
+      atools.matchedRefPositionDecDiff: MatchedRefCoaddDiffPositionTool
+      atools.matchedRefPositionDecDiff.coord_meas: coord_dec
+      atools.matchedRefPositionDecDiff.coord_ref: ref_dec
+      atools.matchedRefPositionDecDiff.produce.plot.xLims: lims_mag_x_full
+      atools.matchedRefPositionDecDiff.produce.plot.yLims: lims_pos_diff_full
+
+      atools.matchedRefPositionDecDiffZoom: MatchedRefCoaddDiffPositionTool
+      atools.matchedRefPositionDecDiffZoom.coord_meas: coord_dec
+      atools.matchedRefPositionDecDiffZoom.coord_ref: ref_dec
+      atools.matchedRefPositionDecDiffZoom.produce.plot.xLims: lims_mag_x_zoom
+      atools.matchedRefPositionDecDiffZoom.produce.plot.yLims: lims_pos_diff_zoom
+      atools.matchedRefPositionDecDiffZoom.produce.metric: NoMetric
+
+      atools.matchedRefPositionDecChi: MatchedRefCoaddDiffPositionTool
+      atools.matchedRefPositionDecChi.coord_meas: coord_dec
+      atools.matchedRefPositionDecChi.coord_ref: ref_dec
+      atools.matchedRefPositionDecChi.produce.plot.xLims: lims_mag_x_full
+      atools.matchedRefPositionDecChi.produce.plot.yLims: lims_pos_chi
+      atools.matchedRefPositionDecChi.compute_chi: true
+
+      python: |
```

Comment:
I also don't like all of this being in here but I think that is personal preference. I think it would look tidier if more of this was in the atool and less in the pipeline.

Reply:
I did what I could to simplify this by consolidating necessary overrides into a single function call in the python block. That function will almost almost always need to be called by anything overriding the task - in obs_subaru, for example, it needs to set the bands for the colour diff plots. I might consider changing the default context from `dc2` to `injection`.

Reply:
The different bands, between obs_subaru and obs_decam for example, should be set in the obs specific config files.